group.yml

```diff
@@ -52,9 +52,9 @@ urls:
   cgit: http://pkgs.devel.redhat.com/cgit
   # temporarily point at 4.9 until 4.10 RHCOS gets rolling
   rhcos_release_base:
-    x86_64: https://releases-rhcos-art.cloud.privileged.psi.redhat.com/storage/releases/rhcos-4.9
-    s390x: https://releases-rhcos-art.cloud.privileged.psi.redhat.com/storage/releases/rhcos-4.9-s390x
-    ppc64le: https://releases-rhcos-art.cloud.privileged.psi.redhat.com/storage/releases/rhcos-4.9-ppc64le
+    x86_64: https://releases-rhcos-art.cloud.privileged.psi.redhat.com/storage/releases/rhcos-4.10
+    s390x: https://releases-rhcos-art.cloud.privileged.psi.redhat.com/storage/releases/rhcos-4.10-s390x
+    ppc64le: https://releases-rhcos-art.cloud.privileged.psi.redhat.com/storage/releases/rhcos-4.10-ppc64le
     aarch64: https://releases-rhcos-art.cloud.privileged.psi.redhat.com/storage/releases/rhcos-4.9-aarch64
```

Comment:
Hi @joepvd 👋🏾 what about aarch64?

Reply:
aarch64 has been building the past few days.

these are only overrides BTW. we only need to override the ones that are not present. now that they are working i'll just remove all of them.